<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Manager</string>
  </property>
  <widget class="QDialogButtonBox" name="LoginButtonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>登录身份</string>
   </property>
   <widget class="QRadioButton" name="FacultyB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>专业(系)B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdmissionsOffice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>招生办</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="FacultyC">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>专业(系)C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="DegreesOffice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>学位办</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FacultyA">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>专业(系)A</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>AdmissionsOffice</tabstop>
  <tabstop>DegreesOffice</tabstop>
  <tabstop>FacultyA</tabstop>
  <tabstop>FacultyB</tabstop>
  <tabstop>FacultyC</tabstop>
  <tabstop>LoginButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>LoginButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoginButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
